lib/rubygems/security/trust_dir.rb

```diff
@@ -3,6 +3,8 @@
 # The TrustDir manages the trusted certificates for gem signature
 # verification.
 
+require 'fileutils'
```

Comment:
Can you move the require inline right before the usage causing issues? Sometimes requiring default gems at the top level causing activation issues with default gems, so we prefer lazily requiring them.

Reply:
Thanks @deivid-rodriguez , https://github.com/rubygems/rubygems/pull/4464/commits/039a290cf1f907c0e6197dac66d1937dee23459b moves inline

Reply:
Nice! Can you rebase and squash everything into a single commit?